foundry/applications/invitationLinks.d.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Game Invitation Links Reference
+ */
+
+declare class InvitationLinks extends Application {
+  /**
+   * @defaultValue
+   * ```typescript
+   * {
+   *   ...super.defaultOptions,
+   *   id: 'invitation-links',
+   *   template: 'templates/sidebar/apps/invitation-links.html',
+   *   title: game.i18n.localize("INVITATIONS.Title"),
+   *   width: 400,
+   * }
+   * ```
+   */
+  static get defaultOptions(): Application.Options;
+
+  /** @override */
+  getData(): { local?: string; remote?: string } | Promise<{ local?: string; remote?: string }>;
```

Comment:
Since this isn't something that would be normally overridden, you should leave the promise off the return.

Reply:
Removed the promise portion of the return.